### PR TITLE
CloudBee Jenkins have 4 segment version

### PR DESCRIPTION
**Pull request by debakarr**
Add one more version segment since Cloudbee have 4 segment version: https://docs.cloudbees.com/docs/cloudbees-jenkins-platform-release-notes/latest/

### CODE REVIEW

**Comment by pbelskiy** [comment]
Thanks for yours two PR!

This package supports Python 3.5 so please rewrite it, also it would be nice to write some unit tests here.

**Comment by debakarr** [comment]
> Thanks for yours two PR!
> 
> This package supports Python 3.5 so please rewrite it, also it would be nice to write some unit tests here.

I think having a method to make NamedTuple is an elegant way for Python 3.5. Another way was to use `__new__` as mentioned here: https://github.com/python/mypy/issues/2683

File: aiojenkins/jenkins.py [hunk 1]
```diff
@@ -18,11 +18,19 @@
 from .plugins import Plugins
 from .views import Views
 
+
 JenkinsVersion = NamedTuple(
-    'JenkinsVersion', [('major', int), ('minor', int), ('patch', int)]
+    "JenkinsVersion",
+    [("major", int), ("minor", int), ("patch", int), ("build", int)],
```
Comment (pbelskiy) [inline aiojenkins/jenkins.py]: single quotes please

File: aiojenkins/jenkins.py [hunk 2]
```diff
@@ -278,11 +286,7 @@ async def get_version(self) -> JenkinsVersion:
 
         versions = header.split('.')
 
-        # no patch version
-        if len(versions) == 2:
-            versions.append('0')
-
-        return JenkinsVersion(*map(int, versions))
+        return make_jenkins_version(*versions)
```
Comment (pbelskiy) [inline aiojenkins/jenkins.py]: I suggest  here to mock aiohttp instead of function mock.

Or I can do it myself it you don't mind.

**Review by pbelskiy — CHANGES_REQUESTED** [review]

File: aiojenkins/jenkins.py (hunk 1 above)
Comment (debakarr) [reply aiojenkins/jenkins.py]: Any plans are there to enable black?

Comment (pbelskiy) [reply aiojenkins/jenkins.py]: I've created an issue about adding black here to check such type of problems. 
https://github.com/pbelskiy/aiojenkins/issues/30

I don't mind if you create PR for this :-)

**Comment by debakarr** [comment]
Any idea on why is coverall failing?

File: tests/test_jenkins.py [hunk 3]
```diff
@@ -151,3 +151,14 @@ def do():
     # just check for no exceptions
     import gc
     gc.collect()
+
```
Comment (pbelskiy) [inline tests/test_jenkins.py:154]: two spaces between functions :-) PEP8

I've just enabled flake8 for tests directory, you can rebase.

**Comment by pbelskiy** [comment]
> Any idea on why is coverall failing?

It's strange, it fails time to time, I will look on it later, but I could accept PR without it, other is okay, and coverage from master after merge also will be okay.

File: aiojenkins/jenkins.py [hunk 4]
```diff
@@ -18,11 +18,19 @@
 from .plugins import Plugins
 from .views import Views
 
+
```
Comment (pbelskiy) [inline aiojenkins/jenkins.py:21]: extra line

**Comment by pbelskiy** [comment]
Fixed in https://github.com/pbelskiy/aiojenkins/releases/tag/v0.7.7

**Pull request merged**